index.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="pt">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href=""><script src=""></script>
+    <title>Womakerscode</title>
+</head>
+
+<body>
+    <img src="images/logo.png" alt="Logo Womakers Code" width="100px">
+    <ul>
+    <li>Home</li>
+    <li>Serviços</li>
+    <li>Portfolio</li>
+    <li>Sobre Mim</li>
+    <li>Contato</li>
+    </ul>
+    
+    <div class="welcome">
```

Comment:
É mais semântico usar a tag section para demarcar as seções

https://www.w3schools.com/tags/tag_section.asp

```suggestion
    <section id="home" class="welcome">
```